src/condor_submit.V6/submit.cpp

```diff
@@ -842,6 +845,12 @@ main( int argc, const char *argv[] )
 			} else if (is_dash_arg_prefix(ptr[0], "help")) {
 				help_info(stdout, --argc, ++ptr);
 				exit( 0 );
+			} else if (is_dash_arg_colon_prefix(ptr[0], "capabilities", &pcolon, 3)) {
+				DashQueryCapabilities = 1;
```

Comment:
See previous comment about `#define`d constants for bitmasks.

Reply:
this one is entirely local to condor_submit, so it can be changed to a mask later if that's needed.